src/generator.js

```diff
@@ -66,11 +81,25 @@ const genCitation = (citeproc, entries, citationId, citationPre, properties = {
  * @param {*} citeproc
  */
 const genBiblioNode = (citeproc) => {
-  const [, bibBody] = citeproc.makeBibliography()
+  const [params, bibBody] = citeproc.makeBibliography()
   const bibliography =
     '<div id="refs" class="references csl-bib-body">\n' + bibBody.join('') + '</div>'
   const biblioNode = htmlToHast(bibliography)
-  return biblioNode
+
+  // Add citekey id to each bibliography entry.
+  const biblioEntries = {}
```

Comment:
I think there's no need for an additional `biblioEntries` object. It seems like the main difference is the id: `inlinebib-` vs `bib-`. We can just use `bib-` in all cases. The inline bib should have an additional class attach to it so the user can target inline vs non-inline relatively easily.

Reply:
Shouldn't HTML ids be unique? It seems important for a11y [purposes](https://www.w3.org/TR/WCAG20-TECHS/H93.html). Classes could be added using the `inlineBibClass` option.

Reply:
Yes, you are right. In this case, I propose just returning `biblioNode` and in the subsequent code that adds the inline bibliography, we can just reassign the `id` from `bib-` to `inlinebib-` over there?

Reply:
Ok, makes sense. I moved `biblioEntries` out in https://github.com/timlrx/rehype-citation/pull/9/commits/3135152b6833a4610dd1ee6696519491798df634. It's now `biblioMap` as a cache to prevent repeated searching for the matching citekey.